app/code/Epam/DI/Api/HtmlGenerator.php

```diff
@@ -0,0 +1,20 @@
+<?php
+
+namespace Epam\DI\Api;
+
+/**
+ * @api
+*/
+interface HtmlGenerator
+{
+    /**
+     * @param CustomInterface $custom
+     * @return void
+     */
+    public function __construct(CustomInterface $custom);
```

Comment:
There is no need to place magic methods into an interface.

Reply:
thanks, fixed!